### DISCUSSION TITLE
Issue: color of the edges according to the nodes

### DISCUSSION THREAD

**Discussion by dagodario** [gephi/gephi#2788] [Q&A] [answered]
Dear community,

Recently, I have been having issues with the "color" mode. When I use the color mode to highlight the nodes along certain parameters, the according edges do not adopt the nodes' color (see screenshot). The box "use node color" in the edges settings is checked.

Has anyone encountered the same issue and found a solution? Am I missing a step?

Thank you very much in advance for your support!

<img width="755" alt="Bildschirm­foto 2023-03-30 um 16 53 27" src="https://user-images.githubusercontent.com/129409478/228876861-121c5a7e-bb9a-4d35-813b-8f6fd4a50bed.png">

**Comment by mbastian** [comment, marked as answer]
Hi @dagodario, 

In Overview, when edges have their own assigned color (i.e. one color for each edge), they take precedence over the node color mode. That is poorly documented I believe. The edges' color is probably imported from your data file. 

You can reset the edge colors by clicking this button. This should erase the underlying color for each edge.

<img width="270" alt="Screenshot 2023-03-30 at 21 17 57" src="https://user-images.githubusercontent.com/234813/228941364-953ae85a-777b-45a9-b531-903b02f9bb50.png">

**Reply by dagodario** [reply]
Dear @mbastian ,

This solved my problem, resetting the colors, then applying the "color mode" on the nodes again.

thank you very much for your help!
